### ISSUE TITLE
Please provide training details

### ISSUE THREAD

**Issue by tearscoco** [EvelynFan/FaceFormer#12]
Hi,

I am trying to train the model from the scratch on vocaset dataset. However, I am not sure whether the training statistics are correct or not since the loss for training got quite small from the beginning.

「(Epoch 1, iteration 115) TRAIN LOSS:0.0000014」

Could you please provide the training details like starting loss, ending loss, number of epochs for model to converge?

Thanks a lot!

**Comment by EvelynFan** [comment]
> Hi,
> 
> I am trying to train the model from the scratch on vocaset dataset. However, I am not sure whether the training statistics are correct or not since the loss for training got quite small from the beginning.
> 
> 「(Epoch 1, iteration 115) TRAIN LOSS:0.0000014」
> 
> Could you please provide the training details like starting loss, ending loss, number of epochs for model to converge?
> 
> Thanks a lot!

Yes. The training loss is quite small. The training details (number of epochs, learning rate, etc.) have been provided in the script "main.py". Using the default parameters in "main.py" can work.

**Comment by tearscoco** [comment]
> > Hi,
> > I am trying to train the model from the scratch on vocaset dataset. However, I am not sure whether the training statistics are correct or not since the loss for training got quite small from the beginning.
> > 「(Epoch 1, iteration 115) TRAIN LOSS:0.0000014」
> > Could you please provide the training details like starting loss, ending loss, number of epochs for model to converge?
> > Thanks a lot!
> 
> Yes. The training loss is quite small. The training details (number of epochs, learning rate, etc.) have been provided in the script "main.py". Using the default parameters in "main.py" can work.

Thanks for quick reply! The loss was quite small and got 0.0000003 after half of the epoches. Can I know what the loss will be when the model converges? And how can I know if the model is well trained or not?

**Comment by EvelynFan** [comment]
> > > Hi,
> > > I am trying to train the model from the scratch on vocaset dataset. However, I am not sure whether the training statistics are correct or not since the loss for training got quite small from the beginning.
> > > 「(Epoch 1, iteration 115) TRAIN LOSS:0.0000014」
> > > Could you please provide the training details like starting loss, ending loss, number of epochs for model to converge?
> > > Thanks a lot!
> > 
> > 
> > Yes. The training loss is quite small. The training details (number of epochs, learning rate, etc.) have been provided in the script "main.py". Using the default parameters in "main.py" can work.
> 
> Thanks for quick reply! The loss was quite small and got 0.0000003 after half of the epoches. Can I know what the loss will be when the model converges? And how can I know if the model is well trained or not?

After the training process is finished, you can visualize the results by running the "render.py" script. From the rendered videos, you can know whether the model is well trained. The decline of the training loss does not necessarily mean the model is well trained. You need to visualize the predictions to judge.

**Comment by tearscoco** [comment]
> 

Instead of visual comparison, do we have any quantitative metrics? Like those error comparisons on Now challenge?

**Comment by EvelynFan** [comment]
> > 
> 
> Instead of visual comparison, do we have any quantitative metrics? Like those error comparisons on Now challenge?

Yes, you can also use the norm on the prediction error as a quantitative metric. For this task, a qualitative evaluation would be better. We focus on the visual quality and the plausibility of the animation.

**Comment by tearscoco** [comment]
> 

Thanks Evelyn! You helped a lot!
